app/assets/stylesheets/geoblacklight/modules/item.scss

```diff
@@ -1,10 +1,32 @@
 #document {
   .document-metadata.dl-invert {
-    dt {
+     dt {
       font-weight: bold;
     }
   }
 
+  #viewer-container {
```

Comment:
My assertion when removing this was that `#viewer-container` is not a child of `#document`. Is that wrong?

Reply:
Yeah. It's a child of #document.

Reply:
document -> document-main-section -> row -> viewer-container